etc/docs/utils.ts

```diff
@@ -0,0 +1,95 @@
+import { createInterface } from "readline";
```

Comment:
similar to Durran's comment, we should probably stick to single quotes for strings, there is a mix and match of them here and in L43